analyses/molecular-subtyping-LGAT/04-LGAT-compile-subtypes.Rmd

```diff
@@ -0,0 +1,304 @@
+path_dx_list <- jsonlite::fromJSON(
+  file.path(subset_dir, 
+            "lgat_subtyping_path_dx_strings.json")
+)
+
+# clinical file
+lgat_manifest <- read_tsv(file.path(root_dir, 
+                                    "data",
+                                    "pbta-histologies-base.tsv"), 
+                          guess_max = 10000) %>%
+  # use inclusion/exclusion criteria from pathology_diagnosis
+  dplyr::filter(str_detect(str_to_lower(pathology_diagnosis),  # Inclusion criteria
+                           paste(path_dx_list$include_path_dx, collapse = "|")),
+                # Exclusion criteria
+                str_detect(str_to_lower(pathology_diagnosis),
+                           paste(path_dx_list$exclude_path_dx, collapse = "|"),
+                           negate = TRUE),
+                # Tumors
+                sample_type == "Tumor",
+                composition == "Solid Tissue") %>%
+  # select columns for interest
+  select(Kids_First_Biospecimen_ID,
+         Kids_First_Participant_ID,
+         sample_id,
+         experimental_strategy,
+         cancer_predispositions)
+
+lgat_wgs <- lgat_manifest %>%
+  filter(experimental_strategy != "RNA-Seq") %>%
+  select( -experimental_strategy, -cancer_predispositions)
+
+lgat_rna <- lgat_manifest %>%
+  filter(experimental_strategy == "RNA-Seq") %>%
+  select( -experimental_strategy, -cancer_predispositions)
+
+all_lgat_matched <- full_join(lgat_wgs, lgat_rna,
+                         by = c("Kids_First_Participant_ID",
+                                "sample_id"),
+                         suffix = c("_DNA", "_RNA")) %>%
+  select(starts_with("Kids_First"), sample_id)
+
+# SNV based annotation
+lgat_snv <- read_tsv(file.path(subset_dir,"LGAT_snv_subset.tsv")) 
+
+# CNV based annotation
+lgat_cnv <- read_tsv(file.path(subset_dir,"LGAT_cnv_subset.tsv")) %>%
+  # To-do add to CNV subset script
+  rename(Kids_First_Biospecimen_ID=lgat_wgs_biospecimen_ids)
+
+# Fusion base annotation
+lgat_fus <- read_tsv(file.path(subset_dir,"LGAT_fusion_subset.tsv")) 
+
+```
+
+
+### LGG, NF1
+This subtype is characterized by germline NF1 variants. We do not have germline data within OpenPBTA, but we do have information about cancer predispositions. This group can be annotated if NF-1 is present within the cancer_predispositions field of the pbta-histologies.tsv file. We should also annotate somatic loss of NF1 via either missense, nonsense mutation or copy loss.
+```{r}
+nf1_snv <- lgat_snv %>%
+  filter(NF1_mut=="Yes") %>%
+  mutate(molecular_subtype = "NF1") %>%
+  select("Kids_First_Biospecimen_ID","molecular_subtype") 
+
+nf1_germline <- lgat_manifest %>% 
+  # to keep non-RNA files only since we will match the RNA 
+  # as Kids_First_Biospecimen_ID_RNA column at a later step
+  filter(experimental_strategy != "RNA-Seq") %>%
+  # keeping "NF1 and NF2" in cancer_predisposition
+  filter(cancer_predispositions %in% c("NF-1",
+                                       "NF-2,Schwannomatosis",
```

Comment:
Reading up on #790 - I don't see why we should be including NF2 here. Did I miss something in an earlier PR?

Reply:
It should only be NF-1, sorry I guess I misread